src/test/java/org/takes/facets/fork/TkMethodsTest.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.facets.fork;
+
+import java.io.IOException;
+import org.apache.commons.lang.RandomStringUtils;
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+import org.mockito.Matchers;
+import org.mockito.Mockito;
+import org.takes.HttpException;
+import org.takes.Take;
+import org.takes.rq.RqFake;
+
+/**
+ * Test case for {@link TkMethods}.
+ * @author Aleksey Popov (alopen@yandex.ru)
+ * @version $Id$
+ * @since 1.0
+ * @checkstyle MagicNumberCheck (500 lines)
+ */
+public class TkMethodsTest {
+    /**
+     * Tests if TkMethods acts on method that is passes to it.
+     * @throws Exception if any error occurs
+     */
+    @Test
+    public final void canCallActOnProperMethods() throws Exception {
+        final String method = RandomStringUtils.random(10);
+        final Take take = Mockito.mock(Take.class);
+        final RqFake propreq = new RqFake(method);
+        new TkMethods(take, method).act(propreq);
+        Mockito.verify(
+            take,
+            Mockito.times(1)
+        ).act(
+                Matchers.argThat(
+                    CoreMatchers.equalTo(propreq)
+                )
+            );
+    }
+
+    /**
+     * Tests if TkMethods throws HttpExcection when acting on unproper method.
```

Comment:
@cyberone Should be "TkMethods can throw HttpException..." see [our coding standards here](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names).